mask_flags.py

```diff
@@ -64,6 +66,13 @@ def parse_helpfull_output(help_output):
     --[no]bool_flag: Flag description
     ''') == {'--some_flag', '--bool_flag', '--nobool_flag'}
 
+assert parse_helpfull_output('''
```

Comment:
do you mind making a new file tests/test_mask_flags.py and testing this and the python variant?

Reply:
Sure.